packages/calcite-components/src/components/date-picker-month-header/date-picker-month-header.tsx

```diff
@@ -70,9 +84,21 @@ export class DatePickerMonthHeader {
    * @internal
    * @readonly
    */
-  // eslint-disable-next-line @stencil-community/strict-mutable -- updated by t9n module
+  // eslint-disable-next-line @stencil-community/strict-mutable
   @Prop({ mutable: true }) messages: DatePickerMessages;
```

Comment:
`mutable` and the `eslint-disable-next-line` comment are not needed since this doesn't implement `T9nComponent`. This might apply to other components.

Reply:
`mutable` is required for testing purposes.

Reply:
🧠  fade!
 
 removed the `mutable ` from messages.